specs/APIReview_TextureStream.md

```diff
@@ -0,0 +1,577 @@
+  /// For example `HTTPS://WWW.ㄓ.COM` will be normalized to
+  /// `https://www.xn--kfk.com` for comparison. So, the input string
+  /// should have a scheme like https://. For example,
+  /// https://www.valid-host.com, http://www.valid-host.com are
+  /// valid origins but www.valid-host.com, or https://*.valid-host.com. are not
+  /// valid origins. If invalid origin is provided, the API will return an error
+  /// of E_INVALIDARG.
+  /// getTextureStream() will fail unless the requesting frame's origin URI is
+  /// added to the allowed origins.
+  /// If `value` is TRUE, then the origin will also be added to WebTexture's
+  /// allowed origin.
+  HRESULT AddAllowedOrigin([in] LPCWSTR origin, [in] BOOL value);
+  /// Remove added origin, which was added by AddAllowedOrigin.
+  /// The allowed or disallowed origins will take effect only when Javascript
+  /// request a streaming. So, once the streaming started, it does not stop
+  /// streaming.
+  HRESULT RemoveAllowedOrigin([in] LPCWSTR origin);
+  /// Listens for stream requests from the Javascript's getTextureStream call
+  /// for this stream's id. It is called for the first request only, and will
+  /// not be called with subsequent requests of same stream id from any pages
+  /// in the middle of request handling or after it returns success.
+  /// The request is regarded as success only when the host provides the stream,
+  /// Present API call, within 10s after being requested.
+
+  /// The Texture stream becomes 'Started' state once it starts sending a texture
+  /// until it calls Stop API or receives 'Stopped' event.
+  HRESULT add_StartRequested(
+      [in] ICoreWebView2StagingTextureStreamStartRequestedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+  /// Remove listener for StartRequest event.
+  HRESULT remove_StartRequested(
+      [in] EventRegistrationToken token);
+  /// Listen to stop stream request once the stream started.
+  /// It is called when user stop all streaming requests from
+  /// the renderers (Javascript) or the host calls the Stop API. The renderer
+  /// can stream again by calling the streaming request API.
+  /// The renderer cleared all registered TextureBuffers before sending
+  /// the stop request event so that the callback of the next start request
+  /// should register the textures again.
+  /// The event is triggered when all requests for given stream id closed
+  /// by the Javascript, or the host's Stop API call.
+  /// TextureBuffer related API calls after this event will return an error
+  /// of HRESULT_FROM_WIN32(ERROR_INVALID_STATE).
+  HRESULT add_Stopped(
+      [in] ICoreWebView2StagingTextureStreamStoppedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+  /// Remove listener for Stopped event.
+  HRESULT remove_Stopped(
+      [in] EventRegistrationToken token);
+  /// Creates TextureBuffer that will be referenced by the host and the browser.

```

Comment:
Also update Buffer -> Texture and TextureBuffer -> Texture throughout documentation and parameter names